src/addrman.cpp

```diff
@@ -962,6 +970,28 @@ std::optional<AddressPosition> AddrManImpl::FindAddressEntry_(const CAddress& ad
     }
 }
 
+size_t AddrManImpl::size_(std::optional<Network> net, std::optional<bool> in_new) const
```

Comment:
Yes, I had used lower case for consistency. I now changed it to upper case, and added a commit that removes the old `size()` to always use the new `Size()` as suggested by Amiti. I made that an extra commit because it involved renaming a lot of existing call sites.